lib/transports/websocket.js

```diff
@@ -79,7 +79,7 @@ class WebSocket extends Transport {
     const send = data => {
       if (this.perMessageDeflate) {
         const len =
-          "string" === typeof data ? Buffer.byteLength(data) : data.length;
```

Comment:
I think that's the correct behavior, we get the number of bytes in both cases:

- string: [`Buffer.byteLength(string)`](https://nodejs.org/api/buffer.html#buffer_static_method_buffer_bytelength_string_encoding)
- buffer: [`buffer.length`](https://nodejs.org/api/buffer.html#buffer_buf_length)

What do you think?

Reply:
Ahh, sorry, i've misunderstood it.